project_issue_reassign/README.rst

```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+The Responsible (``user_id``) field is made read only, and can instead be
```

Comment:
Last time I looked forms didn't behave well when the same field was added twice.
But the wizard has additional advantages: you can make several changes on a single write operstion: now it's user_id and project_id, but I would like to extend that. For example, a message (think of GitHub's "Comment and Close").